Description: Document process field limit in error message and manual page
Author: Johannes Truschnigg <johannes@truschnigg.info>
Last-Update: 2012-11-23
Status:  applied upstream

diff --git a/man/inittab.5 b/man/inittab.5
index e1d739d..80aa084 100644
--- a/man/inittab.5
+++ b/man/inittab.5
@@ -66,7 +66,7 @@ a `+' character,
 .B init 
 will not do utmp and wtmp accounting for that process.  This is needed for 
 gettys that insist on doing their own utmp/wtmp housekeeping.  This is also 
-a historic bug.
+a historic bug. The length of this field is limited to 127 characters.
 .\"}}}
 .PP
 The \fIrunlevels\fP field may contain multiple characters for different
diff --git a/src/init.c b/src/init.c
index 27532ad..f8bf891 100644
--- a/src/init.c
+++ b/src/init.c
@@ -1314,7 +1314,7 @@ void read_inittab(void)
 	if (rlevel && strlen(rlevel) > 11)
 		strcpy(err, "rlevel field too long (max 11 characters)");
 	if (process && strlen(process) > 127)
-		strcpy(err, "process field too long");
+		strcpy(err, "process field too long (max 127 characters)");
 	if (action && strlen(action) > 32)
 		strcpy(err, "action field too long");
 	if (err[0] != 0) {
